micro_dl/cli/image_inference.py

```diff
@@ -229,33 +261,61 @@ def run_prediction(args, gpu_ids, gpu_mem_frac):
                     np.save(file_name, im_pred, allow_pickle=True)
                 else:
                     raise ValueError('Unsupported file extension')
+
+                # assuming target and predicted images are always 2D for now
+                # Load target
+                meta_idx = aux_utils.get_meta_idx(
+                    frames_meta,
+                    time_idx,
+                    target_channel,
+                    slice_idx,
+                    pos_idx,
+                )
+                # get a single row of frame meta data
+                test_frames_meta_row = frames_meta.loc[meta_idx]
+                im_target = preprocess_imstack(
+                    frames_metadata=frames_meta,
+                    input_dir=args.image_dir,
+                    depth=1,
+                    time_idx=time_idx,
+                    channel_idx=target_channel,
+                    slice_idx=slice_idx,
+                    pos_idx=pos_idx,
+                )
+                im_target = image_utils.crop2base(im_target)
+                if data_format == 'channels_first':
+                    # Change dimension order to zyx (3D) or channel first (2D)
+                    im_target = np.transpose(im_target, [2, 0, 1])
+                # add the batch dimension
+                im_target = im_target[np.newaxis, ...]
+                if len(im_target.shape) < len(im_stack.shape):
+                    im_target = im_target[np.newaxis, ...]   
+                metric_vals = model.evaluate(x=im_stack, y=im_target)
+                for metric, metric_val in zip([loss]+metrics, metric_vals):
+                    test_frames_meta_row[metric] = metric_val
+
+                test_frames_meta = test_frames_meta.append(test_frames_meta_row, ignore_index=True)
```

Comment:
We try to avoid line lengths over 80, so can I ask you to:
```
test_frames_meta = test_frames_meta.append(
    test_frames_meta_row, 
    ignore_index=True,
)
```

Reply:
Sure. I'll keep the line short